### ISSUE TITLE
[Bug]: <title> youtube music is not connecting 

### ISSUE THREAD

**Issue by str33tmeat** [XeroxDev/YTMD-StreamDeck#110]
### Consent

- [X] I have searched the existing issues and verified that there is no open issue for the same subject.
- [X] I understand that the YTMDesktop Elgato Plugin has NO affiliation with Google, YouTube or the YTMDesktop application.

### Current Behavior

it was working since I downloaded the plugin, I didnt even need to touch any settings and it worked but since yesturday it just stopped working. The YTM app I'm using doesnt even have the botton that gives you the host numbers like vidoes tell me. 

### Expected Behavior

I expecte it to just keep working like how it was.

### Steps To Reproduce

1.I got back onto my pc and it gave me the warning symbole on the play-pause botton.
2. I havent gotten it to work ever since I got back onto my pc yesturday.

### OS

Windows

### OS Version

10

### Plugin Version

v2.1.0

### YTMDesktop Version

v3.3.2

### How did you download the software?

Elgato Store

### Anything else?

![YTM bs](https://github.com/XeroxDev/YTMD-StreamDeck/assets/162455892/48e7eab9-33cb-48e7-8279-e461a0ac757b)
idk if it helps but its the picutre of the remot controle botton not being there in the app to connect the stream deck to it.

**Comment by XeroxDev** [comment]
Hello and thank you for opening this ticket.

For some reasons your YTMDesktop version doesn't even look similar to the one I'm using

![image](https://github.com/XeroxDev/YTMD-StreamDeck/assets/10864620/fa83e49f-b256-483c-9853-38192ffbe1ef)

Which one do you use? Because `v3.3.2` of YTMDesktop isn't known to be as well. The latest one is `v2.0.2`.

**Comment by XeroxDev** [comment]
After some research I found out that you're using a different kind of application and I've to inform you that this version is not supported by me. The only reason why it worked in the past is that it's probably a fork / clone of the original one.

For now I don't have the intention to support other kinds of youtube music apps except YTMDesktop, therefore I gonna /close this ticket.

If you still have questions or you think this is an error, feel free to comment or even reopen this issue.

**Comment by str33tmeat** [comment]
Hi thanks for informing me that I'm using the wrong version but the version that is supported by you is not available to me. When I click on the YTMDesktop in the description but it brings me to an error screen stating that it so no where to be found. That is why I'm using this one right now The original version I was using before this one was working fine but I deleted it after it stopped working thinking it would fix the problem but when I went to go find it again it was gone.
